addons/csw/functions/fnc_deployTripod.sqf

```diff
@@ -0,0 +1,55 @@
+/*
+ * Author: TCVM
+ * Deploys the tripod
+ *
+ * Arguments:
+ * 0: Unit <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [player] call ace_csw_fnc_deployTripod
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+params ["_player"];
+
+[{
+    params["_player"];
+    // Remove the tripod from the launcher slot
+    private _secondaryWeaponClassname = secondaryWeapon _player;
+    _player removeWeaponGlobal (secondaryWeapon _player);
```

Comment:
Does this also remove magazines loaded in the secondary weapon? If so, re-adding any removed magazines may need to be handled at line 49.

Reply:
Since the secondaryWeapon in this case is the tripod, there should be zero ammo in the secondaryWeapon. I think it's not needed, but if you really think its should exist I will add it